mgmtsystem_action/models/mgmtsystem_action.py

```diff
@@ -19,11 +33,14 @@ def _default_owner(self):
         """Return the user."""
         return self.env.user
 
+    @api.model
     def _default_stage(self):
         """Return the default stage."""
-        return self.env['mgmtsystem.action.stage'].search(
-            [('is_starting', '=', True)],
-            limit=1)
+        return (
+            self.env.ref('mgmtsystem_action.stage_draft', False) or
```

Comment:
Why change the default behaviour ? I suppose we should be able not to use stage_draft as default (is_starting=True).